packages/pensieve/install

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -x
+set -e
+
+apt-get update -qq
+apt-get install -y pdf2svg
```

Comment:
combine the install: `apt-get install -y pdf2svg librsvg2-bin`

Reply:
done, thanks.